### PR TITLE
fix invisible charts

### PR DESCRIPTION
**Pull request by osdnk**
Of course, so I observed that previously we were using hack `useReactiveValue`, but I noticed that in the new version of reanimated it just works as expected.
To make it workable then (1 month ago) then I was setting the value initially to `null`.
However, probably because of some optimizations in the new version this `null` value got derived to the component (as the width of the line) before overriding it with the correct value making the whole chart invisible.

### CODE REVIEW

File: src/react-native-animated-charts/src/helpers/useReactiveSharedValue.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-import { useEffect } from 'react';
```
Comment (osdnk) [inline src/react-native-animated-charts/src/helpers/useReactiveSharedValue.js:1]: looks like it's no longer needed with the new version of reanimated

**Review by mikedemarais — APPROVED** [review]
code lgtm

can amigo @skylarbarrera plz confirm it does what it’s supposed to 🙏🧖‍♂️

**Review by jinchung — APPROVED** [review]
🍡

**Comment by christianbaroni** [comment]
🌈💯🌈

File: ios/Podfile.lock [hunk 2]
```diff
@@ -1023,4 +1023,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 2b38dc4e50f6b19e7a7db1e41739f3209e9d05e6
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.10.0
```
Comment (skylarbarrera) [inline ios/Podfile.lock:1026]: seen this bouncing back and forth between versions

**Review by skylarbarrera — APPROVED** [review]
this looks good to go!

File: ios/Podfile.lock (hunk 2 above)
Comment (osdnk) [reply ios/Podfile.lock:1026]: I dont know which one is right

Comment (osdnk) [reply ios/Podfile.lock:1026]: I'll make it fixed to .10 later

**Pull request merged**